Member.py

```diff
@@ -0,0 +1,37 @@
+import test_helpers
+
+
+class Member(object):
+    def __init__(self, member_id: int):
+        self._member_id = member_id
+        self._veteran = False
+        self._tenured = False
+        self._age = test_helpers.lookup_age(self._member_id)
+        assert self._age >= 0, "You must be born to enter the movie"
+        assert self._age <= 124, "You can't be the oldest person alive"
+
+    def get_age_fee(self) -> int:
+        """Return base fee based on age"""
+        if self._age <= 18:
+            base_fee = 6
+        elif self._age >= 65:
+            base_fee = 9
+        else:
+            base_fee = 20
+        return base_fee
+
+    def veteran_status(self) -> bool:
```

Comment:
you could rename these to be a little more readable, like "is_veteran" and "has_tenure"